assets/resources/scss/_variables.scss

```diff
@@ -1,7 +1,8 @@
 :root {
   --primary-color: #fff;
```

Comment:
As I don't really know which is better to use and as I was consider it, I saw in [00-css.md](https://github.com/Hetachi/FrontEndTraining/blob/master/guides/00-css.md) that is says to try to use CSS4 variables, so that's what I did. And also it is quite nice for development purposes when I can change the var value in browser dev tools to see changes, as opposed to SASS variables which are just replaced with const values after compilation.

Reply:
OK, you can keep it the way it is. Remember this argument for Demo